addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -5,12 +5,21 @@
 from odoo import models, _
 from odoo.tools import html2plaintext, cleanup_xml_node
 
+UBL_NAMESPACES = {
+    'cbc': "urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2",
+    'cac': "urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2",
+}
+
 
 class AccountEdiXmlUBL20(models.AbstractModel):
     _name = "account.edi.xml.ubl_20"
     _inherit = 'account.edi.common'
     _description = "UBL 2.0"
 
+    # EXTENDS account.edi.common
+    def _find_value(self, xpath, tree):
```

Comment:
Why not nsmap in the arguments here?

Reply:
Right, it's probably better to add the default nsmap in the signature rather than in the return
(I also need to move the  #EXTENDS)

Reply:
For the default, I think in general, we define it with the same default as the parent, but just call the super with the right argument (like you did)